ref/general/hardening-intro.adoc

```diff
@@ -43,7 +43,7 @@ To learn more about network vulnerabilities and ways to prevent network intrusio
 ==== Best practices
 
 * *Use a firewall*: Use a firewall to shield production Open Liberty servers so that they don't run in the DMZ. The DMZ is an unsecured network segment that exists between internal and external networks. Firewalls help prevent security intrusions by monitoring network traffic and blocking specific traffic based on security rules.
-* *Enable the `transportSecurity` feature*: The feature:transportSecurity[] feature enables the feature:ssl[] feature to support SSL connections and adds the ability to configure outbound transports. In addition to enabling `transportSecurity`, disable non-SSL ports by setting them to `-1` in the `server.xml` file, and use only certificates that are signed by a certificate authority for public-facing production servers.
+* *Enable the `transportSecurity` feature*: The link:/docs/ref/feature/#transportSecurity.html[Transport Security feature] enables the link:/docs/ref/feature/#ssl.html[SSL feature] to support SSL connections and adds the ability to configure outbound transports. In addition to enabling `transportSecurity`, disable non-SSL ports by setting them to `-1` in the `server.xml` file, and use only certificates that are signed by a certificate authority for public-facing production servers. In addition to enabling the `transportSecurity` feature for Open Liberty, enable SSL in your client and web server. Using SSL connections throughout your network communication flow helps protect your server from network intrusions.
```

Comment:
Change,
"* *Enable the `transportSecurity` feature*: The link:/docs/ref/feature/#transportSecurity.html[Transport Security feature] enables the link:/docs/ref/feature/#ssl.html[SSL feature] to support SSL connections and adds the ability to configure outbound transports. In addition to enabling `transportSecurity`, disable non-SSL ports by setting them to `-1` in the `server.xml` file, and use only certificates that are signed by a certificate authority for public-facing production servers. In addition to enabling the `transportSecurity` feature for Open Liberty, enable SSL in your client and web server. Using SSL connections throughout your network communication flow helps protect your server from network intrusions."
to,
"*Enable the `transportSecurity` feature*: The link:/docs/ref/feature/#transportSecurity.html[Transport Security feature] enables the link:/docs/ref/feature/#ssl.html[SSL feature] to support SSL connections and adds the ability to configure outbound transports. Additionally, disable non-SSL ports by setting them to `-1` in the `server.xml` file, and use only certificates that are signed by a certificate authority for public-facing production servers. Also, enable SSL in your client and web server. Using SSL connections throughout your network communication flow helps protect your server from network intrusions."